File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/factories/PercusionistaFactoryTest.java
package simulacionBandaMusica.factories;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.models.Percusionista;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class PercusionistaFactoryTest {

    @Test
    void createPercusionistaTest() {
        Percusionista Percusionista = PercusionistaFactory.create();
        // Percusionista no nulo, para ver si se han asignado correctamente los datos
        assertNotNull(Percusionista);
    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/factories/PercusionistaFactoryTest.kt
package simulacionBandaMusica.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import simulacionBandaMusica.models.Musico

internal class PercusionistaFactoryTest {
    @Test
    fun createPercusionistaTest() {
        val Percusionista: Musico.Percusionista = PercusionistaFactory.create()
        // Percusionista no nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(Percusionista)
    }
}